CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Fixed an issue where **validate** falsely failed with error `DS108` on descriptions ending with new lines followed by square/curly brackets.
 * Fixed an issue where **graph** commands would not clean their temporary files properly, causing successive commands to fail.
 * Fixed an issue where an error log message changed the terminal color.
+* Added the `GCS_MARKET_BUCKET`, `GCS_MARKET_BUCKET_DEV`, `GCS_MARKET_XPANSE_BUCKET`, `GCS_MARKET_V2_BUCKET`, `GCS_MARKET_XSOAR_SAAS_BUCKET`, `DOCKER_IO`, `CI_PROJECT_ID`, `CI_PROJECT_ID` environment variables.
```

Comment:
If any of these are relevant for users, explain what they're for. Otherwise, add the `Internal:` prefix, and a short description for the usages.